process faults {5: ({5, 2}, 1647777044), 2: ({2, 4}, 3344774249)}
network faults {}
{8: ("14A229DC50AFC063B0ECEE62701088FBECE82D07B127AA7B57B7D973CA8D0F1B", {2, 5, 6, 4, 1, 0, 3}), 9: ("0A916D9344D84E42DE646D3BDBEFBA8428FC9D976BAAF64A27EEC51EC3600CB2", {0, 4, 6, 3, 5, 1, 2}), 7: ("BFF3411DC64C4344FAD8AD9E2D458AE56FB2B4F00EB0C99A709D2396868D517F", {3, 1, 6, 0, 4, 5, 2}), 5: ("837999ECB7BC859418987CF94546B5E4AB9D95BB57A0DFE61A466F5196A05B26", {1, 6, 5, 0, 4, 2, 3}), 10: ("AC096887DF39651C69A6CBCF90E2CBA7AB1F093C267348CE291B3886FB690780", {0, 6, 2, 3, 4, 5, 1}), 6: ("CC06059DC88830CB086803D4FF089BBAD72AF46653775A278D9A5BE1DC1D14FC", {3, 4, 0, 2, 1, 6, 5})}
7
reason: all committed
done!
